doc/INSTALL.md

```diff
@@ -0,0 +1,71 @@
+# pick_ik : Installation
+
+## Install from binaries
+
+COMING SOON! Once a release of `pick_ik` is available, you should be able to get this package with, e.g.,:
+
+```
+sudo apt install ros-humble-pick-ik
+```
+
+---
+
+## Install from source
+
+1. Clone this repository in a Colcon workspace.
+
+```shell
+cd /path/to/your/workspace/src
```

Comment:
Consider using a `COLCON_WS` variable like in the MoveIt install instructions or other tutorials